EpikCord/__init__.py

```diff
@@ -2305,6 +2305,63 @@ def latency(self):
     def average_latency(self):
         return sum(self.latencies) / len(self.latencies)
 
+    def add_task(self, task: Callable, interval: Optional[int] = None, **kwargs):
```

Comment:
`Callable` should be precised as `Callable[[...], None]`

Reply:
did not know that, excuse me for my dumbness